### DISCUSSION TITLE
[Hardware] USB-C?

### DISCUSSION THREAD

**Discussion by hahalolha** [joeycastillo/Sensor-Watch#99] [Ideas]
Might be impossible due to case interference however, would be worth investigation. One method to incorporate USB-C could be to use the PCB as the 'centre'/contacts of the connector. Another option is a recessed/inboard connector.

Motivation is: Micro USB is dead and USB C is generally better in durability and future compatibility.

**Comment by bademux** [comment]
while I totally agree with you, lets just assume that due to some technical limitation SensorWatch has  some proprietary connector :)
And it is big "coincidence" that it is compatible with micro usb (and we grateful for that)

**Comment by joeycastillo** [comment]
The reason for Micro B instead of C is that USB C requires two 5.1k pull-down resistors in order to receive power. If you look at the board, there’s really no place to put them.
